pkg/cli/node.go

```diff
@@ -467,12 +470,34 @@ func runDecommissionNodeImpl(
 		}
 
 		if !anyActive && replicaCount == 0 {
-			// We now mark the nodes as fully decommissioned.
-			req := &serverpb.DecommissionRequest{
+			// We now drain the nodes in order to close all SQL connections.
+			// Note: iteration is not necessary here since there are no remaining leases
+			// on the decommissioning node after replica transferral.
+			for _, targetNode := range nodeIDs {
+				if targetNode == serverCfg.IDContainer.Get() {
+					// Skip the draining step for the node serving the request, if it is a target node.
+					fmt.Fprintf(stderr, ""+
```

Comment:
[nit] Do we need the `""+`. Instead of "this node", perhaps include the node ID itself given we have it?

Reply:
Fixed.